docs/runbooks/HPPSharingPoolPathWithOS.md

```diff
@@ -1,34 +1,51 @@
+<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
+
 # HPPSharingPoolPathWithOS
 
 ## Meaning
 
-The hostpath-provisioner (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
-If HPP is sharing a filesystem with other critical components (kubelet / OS) its possible that HPP PVs will cause node disk pressure.
+This alert fires when the hostpath provisioner (HPP) is sharing a file system with other critical components, such as `kubelet` or the operating system (OS).
+
+HPP dynamically provisions hostpath volumes to provide storage for persistent volume claims (PVCs). In the described scenario, HPP persistent volumes might cause node disk pressure.
+
+
 ## Impact
 
-Possible degraded node operation.
+The affected node might have degraded performance and stability.
 
 ## Diagnosis
 
-- Check which HPP pool is being shared with OS by looking at DaemonSet logs
-	```bash
-	export HPP_NAMESPACE="$(kubectl get deployment -A | grep hostpath-provisioner-operator | awk '{print $1}')"
-	```
+Check which HPP pool is being shared with the OS by reviewing the DaemonSet logs.
+
+1. Configure the `HPP_NAMESPACE` environment variable:
+```bash
+$ export HPP_NAMESPACE="$(kubectl get deployment -A | grep hostpath-provisioner-operator | awk '{print $1}')"
+```
 
-- Find CSI DaemonSet pods
-	```bash
-	kubectl -n $HPP_NAMESPACE get pods | grep hostpath-provisioner-csi
-	```
+2. Find the Container Storage Interface (CSI) DaemonSet pods:
```

Comment:
DaemonSet > daemon set

This is an OpenShift docs style requirement. We use lower case with spaces for general references and camel caps for object references.

I suggest changing the lead-in sentence: "Obtain the status of the `hostpath-provisioner-csi` daemon set pods:"